src/components/Header/Nav/SubNavLink/index.js

```diff
@@ -0,0 +1,63 @@
+// @flow
```

Comment:
this has no siblings? should it be an `index`?

Reply:
should this be called `more` or similar? we also ought to be able to merge this veggie burger i think, and maybe call them subnav toggle or somesuch? then it would render as the 'more' link or burger button depending on breakpoint?

Reply:
Yep, definitely doesn't need to be an `index`. I'll look at creating a single more/veggie component too!

Reply:
The bug in emotion detailed at https://github.com/emotion-js/emotion/issues/593 is preventing me from consolidating `VeggieBurger` and `SubNavLink` as merging the 2 into a single styled component requires nested pseudo selectors inside media-queries. 😞